### ISSUE TITLE
progress indicator

### ISSUE THREAD

**Issue by icy** [ingydotnet/git-subrepo#473]
Hi guys,

I need to clone a very huge repository with `git subrepo`, and the command doesn't show any progress. Is that possible to have progress indicator with `subrepo` command? 

Thanks

**Comment by admorgan** [comment]
I like this idea. Do you know of another git tool that shows progress? I as ask this so I can see how they did it so I can figure out how to adapt it to our tool.

**Comment by icy** [comment]
@admorgan The known `git clone` command has some information already. For example:

```
$ git clone git@github.com:my_org/uluru
Cloning into 'uluru'...
remote: Enumerating objects: 256, done.
remote: Counting objects: 100% (256/256), done.
remote: Compressing objects: 100% (161/161), done.
Receiving objects:   5% (7056/134935), 1.89 MiB | 127.00 KiB/s  
```

This is what I get with `git subrepo clone`

```
$ git subrepo clone git@github.com:my_org/uluru --verbose
* Assert that working copy is clean: /home/my_org/projects/incubator
* Check for worktree with branch subrepo/uluru
* Determine the upstream head branch.
  * Fetch the upstream: git@github.com:my_org/uluru (master).
# nothing else here...
```

**Comment by admorgan** [comment]
Good point, I wonder if I can force fetch to display what it normally displays. I can look into this some time next week. If you can get a PR up on the release/0.4.2 branch I would be happy to accept it.

**Comment by icy** [comment]
@admorgan I've found the `--quiet` option is used here https://github.com/ingydotnet/git-subrepo/blob/a04d8c2e55c31931d66b5c92ef6d4fe4c59e3226/lib/git-subrepo#L683 .

Is there any reason why that's off by default? I may send a patch to turn it on, but that would change users' experience a bit. To me, it's fine as I'd expect the same verbose level for two commands (git clone, git subrepo clone)

Thanks

**Comment by icy** [comment]
It's more complicated than I thought. The `RUN` method actually hides everything, unless specified with `debug` mode:

https://github.com/ingydotnet/git-subrepo/blob/a04d8c2e55c31931d66b5c92ef6d4fe4c59e3226/lib/git-subrepo#L1802
